packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -764,6 +767,34 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     detachFiber(current);
   }
 
+  function commitProfileWork(finishedWork: Fiber): void {
+    switch (finishedWork.tag) {
+      case Profiler: {
+        if (enableProfileModeMetrics) {
+          finishedWork.pendingProps.onRender.call(
```

Comment:
```js
let onRender = finishedWork.pendingProps;
onRender(...);
```

Reply:
Lint complains about that form (`no-useless-call`) but I can add an ignore rule if you prefer it.

Reply:
I think @sebmarkbage means we need `memoizedProps`, not `pendingProps`?

Reply:
Oh, haha. I misread that 😄

Reply:
It’s two issues. We should use memoizedProps and avoid .call

Reply:
Already using memoized.

Happy to avoid using .call - but I'd like to know why. 😄

Reply:
Nevermind. It came to me while I was brushing my teeth that I was misinterpreting your statement. I was assuming you were considering something like a performance angle, which I didn't understand. I assume you are considering cases where the callback might be an instance on a logger class or something.

Anyway, this was just me being silly. I've made the change.